doozerlib/rhcos.py

```diff
@@ -9,18 +9,73 @@
 from urllib.error import URLError
 from doozerlib.util import brew_suffix_for_arch, isolate_el_version_in_release
 from doozerlib import exectools
-from doozerlib.model import Model, Missing
+from doozerlib.model import ListModel, Model
 from doozerlib import brew
 
 RHCOS_BASE_URL = "https://rhcos-redirector.apps.art.xq1c.p1.openshiftapps.com/art/storage/releases"
+default_primary_container = dict(  # historically the only RHCOS container
+    name="machine-os-content",
+    build_metadata_key="oscontainer",
+    primary=True)
 
 
-def rhcos_content_tag(runtime) -> str:
```

Comment:
turns out nothing references this; we may need to resurrect it somehow when RHCOS is building for rhel8 and rhel9.